server_test.go

```diff
@@ -166,13 +167,9 @@ func TestLinkGetFoundJSON(t *testing.T) {
 
 	require.Equal("abc", jsonResponse["id"])
 	require.Equal("https://example.com", jsonResponse["url"])
-	require.Equal("yesterday", jsonResponse["@timestamp"])
 
-	require.ObjectsAreEqual(jsonResponse, map[string]string{
-		id:           "abc",
-		url:          "https://example.com",
-		"@timestamp": "yesterday",
-	})
+	_, err = time.Parse(time.RFC3339, jsonResponse["@timestamp"])
```

Comment:
This is great to know: `time.RFC3339`. Interesting that it even has parse layouts as a constant